app/src/test/java/org/pesmypetcare/mypetcare/controllers/community/TestTrGetGroupImage.java

```diff
@@ -0,0 +1,37 @@
+package org.pesmypetcare.mypetcare.controllers.community;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.httptools.utilities.DateTime;
+import org.pesmypetcare.mypetcare.features.community.groups.Group;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.mypetcare.services.StubCommunityService;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Albert Pinto
+ */
+public class TestTrGetGroupImage {
+    private User user;
+    private Group group;
+    private TrGetGroupImage trGetGroupImage;
+
+    @Before
+    public void setUp() {
+        user = new User("John Doe", "johndoe@gmail.com", "1234");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "John Doe" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5529921)